test/validation/custom.validation.spec.ts

```diff
@@ -99,4 +103,13 @@ describe('tests for custom Date fields', () => {
   test('should throw error for checkSecondNationality when nationality same as second nationality', () => {
     expect(() => custom.checkSecondNationality("Zamunda", "Zamunda", { sameError: errors.sameError })).toThrowError(errors.sameError);
   });
+
+  test("should test checkPublicRegisterJurisdictionLength is not triggered when register field not selected", () => {
```

Comment:
i would change the name of the tests so they are similar to the others eg.

"should return truthy value for checkPublicRegisterJurisdictionLength when register field not selected"

"should throw error for checkPublicRegisterJurisdictionLength when register field is selected"